### PR TITLE
Backend threading to avoid commit conflicts (+ website updates)

### PR DESCRIPTION
**Pull request by jannisborn**
INFO:__main__:======Location differ joblog from 2023-12-21 22:30:04=======
INFO:googlemaps.client:API queries_quota: 60
ERROR:pennyme.pennycollector:Geolocation failed for: Unknown Location 	 sub: , Arco
INFO:__main__:Boot Hill Museum is available again
INFO:__main__:Location Kansas (45/125): Changes in 1/46 machines found.
INFO:__main__:
 Result: 1 changes, 1 new machines found and 0 machines retired
ERROR:__main__:Found 5 problems that require manual intervention
INFO:__main__:======Location differ completed at 2023-12-22 00:19:35=======

### CODE REVIEW

**Comment by NinaWie** [comment]
New machine 7011 named Candy Edcentures submitted.

**Comment by jannisborn** [comment]
INFO:__main__:======Location differ joblog from 2023-12-22 22:30:04=======
INFO:googlemaps.client:API queries_quota: 60
INFO:__main__:Dolly Parton's Stampede Dinner & Show is available again
INFO:__main__:134/177: Found machine to be added: Buc-ee's
INFO:__main__:Location Missouri (22/125): Changes in 2/177 machines found.
ERROR:pennyme.pennycollector:Geolocation failed for: Unknown Location 	 sub: , Arco
INFO:__main__:68/90: Found machine to be added: Pillow's Funtrackers
INFO:__main__:Location New Mexico (40/125): Changes in 1/90 machines found.
INFO:__main__:
 Result: 3 changes, 3 new machines found and 0 machines retired
ERROR:__main__:Found 5 problems that require manual intervention
INFO:__main__:======Location differ completed at 2023-12-23 00:19:20=======

File: backend/app.py [hunk 1]
```diff
@@ -220,26 +294,18 @@ def create_machine():
         "geometry": {"type": "Point", "coordinates": location},
         "properties": properties_dict,
     }
-    # If pushing to new branch: set unique branch name
-    # branch_name = f"new_machine_{round(time.time())}"
-    new_machine_id = push_newmachine_to_github(new_machine_entry)
-
-    # Upload the image
-    if "image" not in request.files:
-        return "No image file", 400
```
Comment (NinaWie) [inline backend/app.py:229]: This is not executed anymore, but I think it's okay since we already check in the frontend whether there is an image

**Review by NinaWie — APPROVED** [review]
Thanks for the threading changes! Looks good to me. I added two email changes.

**Pull request merged**